docs/advanced/Concurrency.md

```diff
@@ -4,11 +4,14 @@ In the basics section, we saw how to use the helper effects `takeEvery` and `tak
 
 In this section we'll see how those helpers could be implemented using the low-level Effects.
 
+These helpers return a forked task, which creates an attached task that will block execution of the saga that invokes the helper. If you need the helpers to create a detached task, which is what the official redux-saga's implementation of `takeLatest` and `takeEvery` does, then use `spawn` instead. 
+
```

Comment:
sorry, I think I may have confused you.  the takeLatest() used in the sample code I put above was referencing the takeLatest implementation from the documentation found here (https://redux-saga.github.io/redux-saga/docs/advanced/Concurrency.html).  If I use the official redux-saga's takeLatest() or takeEvery(), then it works fine.